.github/workflows/ci.yaml

```diff
@@ -85,7 +85,7 @@ jobs:
         run: |
             set -ex
             sudo apt update
-            sudo apt install python3-dev libldap2-dev libsasl2-dev libssl-dev
+            sudo apt install python3-dev libldap2-dev libsasl2-dev libssl-dev libxmlsec1 libxmlsec1-dev
```

Comment:
We should have a comment here indicating that libxmlsec1 libxmlsec1-dev are only for the experimental integration. In the future we might skip these things if in the future we want to remove them. 
Also the dev docs should indicate this is necessary. What do you think @Dostonbek1